mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -458,11 +459,6 @@ def get_instruction_set_args(self, instruction_set: str) -> T.Optional[T.List[st
     def get_pch_base_name(self, header: str) -> str:
         return os.path.basename(header)
 
-clangcl_color_args: T.Dict[str, T.List[str]] = {
-    'auto': ['-fcolor-diagnostics'],
-    'always': ['-fcolor-diagnostics'],
-    'never': ['-fno-color-diagnostics'],
-}
```

Comment:
Since you added this in a previous commit, might as well just fold the commit into the other